src/sequelize.d.ts

```diff
@@ -684,6 +685,7 @@ export class Sequelize extends SequelizeTypeScript {
 
   static Op: typeof Op;
   static DataTypes: typeof DataTypes;
+  static Validator: typeof validator;
```

Comment:
I'd like to stop exposing everything on the Sequelize constructor but we currently have to for legacy reasons (see comment in `index.js`). I prefer to not type properties that are added on it for legacy reasons to discourage people from accessing them like this